cs-rin-ru-enhanced-mod.user.js

```diff
@@ -640,81 +681,190 @@ setupPageTitle();
 Made by SubZeroPL
 displays preview of first post from topic that mouse cursor points
 */
+function previewElement(element, link, getIndex) {
```

Comment:
Some documentation for parameters would be nice, for example what getIndex does?